<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewAccount</class>
 <widget class="QDialog" name="NewAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>671</width>
     <height>491</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius:50px;  </string>
   </property>
   <widget class="QLineEdit" name="pre_Account">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #f2f2f7;
 border: none;
 border-radius: 8px;
 padding: 10px;
  font-size: 20px;
QLineEdit {
    /* 输入文字样式 */
    font-size: 20px;                    /* 输入文字变大 */
    font-family: &quot;Microsoft YaHei&quot;, &quot;Segoe UI&quot;, Arial;
    color: #333333;
    padding: 10px 15px;
    border: 2px solid #cccccc;
    border-radius: 6px;
    background-color: white;
    min-height: 44px;
}

/* 提示文字（placeholder）样式 */
QLineEdit::placeholder {
    font-size: 20px;                    /* 提示文字也变大 */
    color: #999999;
    font-style: italic;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pre_Code">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>229</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #f2f2f7;
 border: none;
 border-radius: 8px;
 padding: 10px;
  font-size: 20px;
QLineEdit {
    /* 输入文字样式 */
    font-size: 20px;                    /* 输入文字变大 */
    font-family: &quot;Microsoft YaHei&quot;, &quot;Segoe UI&quot;, Arial;
    color: #333333;
    padding: 10px 15px;
    border: 2px solid #cccccc;
    border-radius: 6px;
    background-color: white;
    min-height: 44px;
}

/* 提示文字（placeholder）样式 */
QLineEdit::placeholder {
    font-size: 20px;                    /* 提示文字也变大 */
    color: #999999;
    font-style: italic;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sure_Code">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #f2f2f7;
 border: none;
 border-radius: 8px;
 padding: 10px;
  font-size: 20px;
QLineEdit {
    /* 输入文字样式 */
    font-size: 20px;                    /* 输入文字变大 */
    font-family: &quot;Microsoft YaHei&quot;, &quot;Segoe UI&quot;, Arial;
    color: #333333;
    padding: 10px 15px;
    border: 2px solid #cccccc;
    border-radius: 6px;
    background-color: white;
    min-height: 44px;
}

/* 提示文字（placeholder）样式 */
QLineEdit::placeholder {
    font-size: 20px;                    /* 提示文字也变大 */
    color: #999999;
    font-style: italic;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="register_Btn">
    <property name="geometry">
     <rect>
      <x>299</x>
      <y>370</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: black;    /* 黑色背景 */
    color: white;              /* 白色文字 */
    border-radius: 5px;        /* 可选：圆角 */
    padding: 8px 16px;         /* 可选：内边距 */
}</string>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_Btn_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: black;    /* 黑色背景 */
    color: white;              /* 白色文字 */
    border-radius: 5px;        /* 可选：圆角 */
    padding: 8px 16px;         /* 可选：内边距 */
}</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NewAcot_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>168</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    
    background-color: transparent;
    
    /* 文字样式 */
    color: #666666;
    font-size: 20px;
    font-weight: 500;
    font-family: &quot;Microsoft YaHei&quot;, sans-serif;
    
    /* 尺寸和边距 */
    padding: 12px 24px;
    min-width: 120px;
    min-height: 48px;
    
    /* 鼠标光标 */
    cursor: pointer;
    
    
}
QPushButton {
    color: rgb(0,,0,0);  /* 选中时蓝色文字 */
    font-weight: 600;
    border-bottom: 3px solid rgb(0,0,0);  /* 蓝色下划线 */
}</string>
    </property>
    <property name="text">
     <string>注册账号</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
